### PR TITLE
Refactor the entire project

### PR DESCRIPTION
**Pull request by ftshijt**

### CODE REVIEW

File: egs/public_dataset/hts/train.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# /usr/bin/bash
+
```
Comment (ftshijt) [inline egs/public_dataset/hts/train.sh:2]: LGTM, nice job

File: egs/public_dataset/kiritan/train.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  
+fi
+
+if [ $stage -le 4 ]; then
+  # train
+  echo ============================================================================
+  echo "                                train                                     "
+  echo ============================================================================
+
+  cmd="python ../../../bin/train.py -c conf/train_rnn_gs.yaml"
```
Comment (ftshijt) [inline egs/public_dataset/kiritan/train.sh:45]: we may change this part by export Path in path.sh

Comment (PeterGuoRuc) [reply egs/public_dataset/kiritan/train.sh:45]: I will simulate the whole process of train.sh later, including downloading data and preprocessing. Path.sh will be added at that time

Comment (ftshijt) [reply egs/public_dataset/kiritan/train.sh:45]: I'm afraid the data downloading may not be possible ( I remember the dataset need some authentication process)

**Pull request merged**